core_gems.rb

```diff
@@ -11,3 +11,4 @@
 download "tzinfo", "1.2.2"
 download "tzinfo-data", "1.2016.5"
 download "google-protobuf", "3.5.1"
+download "grpc", "1.9.1"
```

Comment:
This wasn't here before, so I'm led to believe another one of the dependencies is pulling in a newer version. Will pinning this guarantee that this is what's being used as opposed to another version pulled in from another dependency?

Reply:
The reason this is new here is because that `grpc` switched to `rake compiler` in the latest release. Then this issue is triggered: https://github.com/rake-compiler/rake-compiler/issues/146.

Reply:
SGTM